### PR TITLE
[sentencepiece] Update port to v0.1.95

### PR DESCRIPTION
**Pull request by JackyYin**
Update sentencepiece from 0.1.82 to 0.1.95.

Successfully built in the following triplet:
- x64-osx
- x64-linux
- x86-windows-static
- x64-windows-static

### CODE REVIEW

**Comment by xavier2k6** [comment]
Why not go with latest v0.1.96?
https://github.com/google/sentencepiece/releases/latest

**Comment by JackyYin** [comment]
Hi @xavier2k6,
because currently I need a `0.1.95` version,
maybe I can open another PR for `0.1.96` after this PR is merged.

**Review by JonLiu1993 — APPROVED** [review]

File: ports/sentencepiece/portfile.cmake [hunk 1]
```diff
@@ -5,12 +5,10 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO google/sentencepiece
-    REF v0.1.82
-    SHA512   669d6a1e86c44587d725b1e93f11b707e510a180dec08afb79268158f5de009cb20ffccc72c501c84f032360e52e53ae227504f3538f59978629433e0d6fcf65
+    REF v0.1.95
+    SHA512 22484cf0311315e25a3184561f4e18b45286ad068bfb722c860e1b44fb16913c96d34723b486fab5e4e1e69f4b620a0e3cff410f56cb6561a67193ebaf565a31
     HEAD_REF master
-    PATCHES rename-version.patch
```
Comment (ras0219-msft) [inline ports/sentencepiece/portfile.cmake:11]: Could you also remove the no longer used patchfile?

https://github.com/microsoft/vcpkg/blob/1021933b268afa63a28378dec8ce85bde0b01d30/ports/sentencepiece/rename-version.patch

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
LGTM after the patch file is removed. Thanks for the PR!

File: ports/sentencepiece/portfile.cmake (hunk 1 above)
Comment (JackyYin) [reply ports/sentencepiece/portfile.cmake:11]: Hello @ras0219 ,
I just removed the patch file.
please check.

**Comment by BillyONeal** [comment]
simdutf should be fixed by https://github.com/microsoft/vcpkg/pull/19987

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Comment by cenit** [comment]
https://github.com/JackyYin/vcpkg/blob/e113e89769c8cc6aab6527e9ab86c0ece6019645/ports/sentencepiece/portfile.cmake#L25

@ras0219-msft 
isn’t this even worse than using an imposed CMAKE_DEBUG_SUFFIX, which is clearly stated as unwanted on vcpkg?
or, let me rephrase: isn’t it better to use CMAKE_DEBUG_SUFFIX instead of hard renaming an artifact after the built?

**Comment by JackyYin** [comment]
Hello @cenit,
Thanks for your feedback.
I think I can fix this in another PR, 
maybe add an OPTIONS `-DCMAKE_DEBUG_POSTFIX=d` and remove line 25 - line 29 as you mentioned.

**Comment by cenit** [comment]
let's discuss with @ras0219-msft because that option is really not wanted here (even too much avoided imho), but again those lines are much worse and they passed the review

**Comment by BillyONeal** [comment]
> let's discuss with @ras0219-msft because that option is really not wanted here (even too much avoided imho), but again those lines are much worse and they passed the review

Some inconsistency between reviewers, sorry. :/

In this case the usual problems with renaming .libs aren't problems because this is an `ONLY_STATIC_LIBRARY` port
